test/CalculatorW3GTest.cpp

```diff
@@ -0,0 +1,144 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   CalculatorGtest.cpp
+ *  @date   Wed May 26 2021
+ *  @brief  Calculator Unit Tests
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include "ICalculatorW3.h"
+#include "SimpleExpressionCalculatorW3Factory.h"
+#include "AddCalculatorW3.h"
+#include "SubtractCalculatorW3.h"
+#include "MultiplyCalculatorW3.h"
+#include "DivideCalculatorW3.h"
+
+using namespace testing;
+using namespace calculator;
+
+/**
+ * @brief Fixture for Simple expression calulcators
+ * 
+ */
+class SimpleExpressionCalculatorW3Test : public ::testing::Test 
+{
+    protected:
+        AddCalculatorW3 addCalculator = AddCalculatorW3(10,10);
+        SubtractCalculatorW3 subtractCalculator = SubtractCalculatorW3(10, 10);
+        MultiplyCalculatorW3 multiplyCalculator = MultiplyCalculatorW3(10, 10);
+        DivideCalculatorW3 divideCalculator = DivideCalculatorW3(10, 10);
```

Comment:
We should each operation inside of the unit test, I understand we want to reuse them across the all UT but it makes it hard to read and keep track of each one of them.
Also each unit test should handle assert a single operation